ui/admin/tests/e2e/tests/auth-method-ldap.spec.js

```diff
@@ -135,6 +125,14 @@ test('Set up LDAP auth method @ce @ent @docker', async ({ page }) => {
     await createRole(page);
     await addPrincipalToRole(page, ldapManagedGroupName);
   } finally {
+    await authenticateBoundaryCli(
+      process.env.BOUNDARY_ADDR,
+      process.env.E2E_PASSWORD_AUTH_METHOD_ID,
+      process.env.E2E_PASSWORD_ADMIN_LOGIN_NAME,
+      process.env.E2E_PASSWORD_ADMIN_PASSWORD,
+    );
+    let orgs = JSON.parse(execSync('boundary scopes list -format json'));
```

Comment:
None blocking comment: these 2 vars `orgs` and `org` can be `const`, because they are not redefined.

Reply:
I'll update